lib/cartopy/tests/test_polygon.py

```diff
@@ -192,8 +192,8 @@ def test_tiny_point_between_boundary_points(self):
         area = projected.area
         # Before fixing, this geometry used to fill the whole disk. Approx
         # 1.2e14.
-        self.assertTrue(2.4e9 < area < 2.5e9,
-                        msg='Got area {}, expecting ~2.4e9'.format(area))
+        self.assertTrue(81330 < area < 81340,
+                        msg='Got area {}, expecting ~81336'.format(area))
```

Comment:
Does changing the boundary really introduce a change of 5 orders of magnitude here?